apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -338,6 +338,36 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
         }
     };
 
+    /**
+     * Modifies the grant type label string value. For now we only concatenate
+     * some extra value to the `implicit` field. If you need to do the same for
+     * other checkboxes label then add a condition and bind the values.
+     *
+     * @param value {string} checkbox key {@link TEMPLATE_WISE_ALLOWED_GRANT_TYPES}
+     * @param label {string} mapping label for value
+     */
+    const modifyGrantTypeLabels = (value: string, label: string): string => {
+        if (value === "implicit") {
+            return `${ label } (not recommended)`;
+        }
+        return label;
+    };
+
+    /**
+     * Generates a string description/hint for the target {@code value} checkbox.
+     * {@see TEMPLATE_WISE_ALLOWED_GRANT_TYPES} for different types.
+     *
+     * @param value {string}
+     */
+    const getGrantTypeHintDescription = (value: string): string => {
+        switch (value) {
+            case "implicit":
```

Comment:
If moved to a constant above, we could use the same here.

Reply:
This also has been addressed in commit f4a3ac7